lntest/harness.go

```diff
@@ -1013,6 +1013,12 @@ func (h *HarnessTest) OpenChannel(alice, bob *node.HarnessNode,
 	// First, open the channel without announcing it.
 	cp := h.OpenChannelNoAnnounce(alice, bob, p)
 
+	// We need to make sure we wait here for 1 second so the ChanUpdate
+	// after 6 confirmations are reached is not considered stale. This
+	// ChanUpdate is important because it unsets the __dont_forward__ bit
+	// signaling the broadcasting to the network.
+	time.Sleep(time.Second)
```

Comment:
I don't think we need to sleep here? If the channel updates have diffs they shouldn't be considered as stale?

Reply:
So the problem which manifests in the itests lies here:
https://github.com/lightningnetwork/lnd/blob/14a6f539717232c100e71447d7bcf7610c210559/discovery/gossiper.go#L2721-L2732

our timestamp granularity is 1 second, now I introduced the Change where we send a ChanUpdate after 3 blocks(if default value is active) but also after 6 blocks with the __dont_forward__ bit unset, because in the itests blocks are mined very fast, we hit exactly the case where the ChanUpdates have the exact same timestamp. We did not see this effect prior to this change tho. So to catch this case in the itests, we have to include a check in the `isStaleEdgePolicy` that ChanUpdates which are different but have the same timestamp are not considered stale ? Wondering if this could have some side effects ?